assets/script.js

```diff
@@ -82,4 +82,20 @@ if (document.location.pathname.indexOf('fullcpgrid') > -1) {
 // Current year of footer
 const currentYearSpan = document.getElementById('currentYear')
 const currentYear = new Date().getFullYear()
-currentYearSpan.innerText = currentYear
\ No newline at end of file
+currentYearSpan.innerText = currentYear
+
+// Search bar 
+function search_card() {
+  let input = document.getElementById('searchbar').value
```

Comment:
1. Naming is very important. As you can see we used `input` above in the variable that selects the input. This here is the value we are taking so we should name it `value`
2. It's good practice to take the value directly from the event object fired by the event listener `event.currentTarget.value`
3. We can also transform the value to lowercase directly here saving one extra step.

` const value = event.currentTarget.value.toLowerCase()`